Project.toml

```diff
@@ -1,10 +1,12 @@
 name = "IntervalArithmetic"
 uuid = "d1acc4aa-44c8-5952-acd4-ba5d80a2a253"
 repo = "https://github.com/JuliaIntervals/IntervalArithmetic.jl.git"
-version = "0.20.8"
+version = "1.0.0"
```

Comment:
I think it was agreed to have this as 0.21.0 version...; see [this comment](https://github.com/JuliaIntervals/IntervalArithmetic.jl/issues/557#issuecomment-1588578133).